src/Stratis.Bitcoin.IntegrationTests/NodeSyncTests.cs

```diff
@@ -304,35 +251,5 @@ public void Pow_MiningNodeWithOneConnection_AlwaysSynced()
                 TestHelper.WaitForNodeToSync(nodes.ToArray());
             }
         }
-
-        [Fact]
-        [Trait("Unstable", "True")]
-        public void Pos_NodesCanConnect_AndSync_AndMineBlocks()
```

Comment:
Is this unstable?

Reply:
Yes, it failed for me

Reply:
We can remove this yes as we have many tests that has POS nodes connecting to each other.